enterprise/frontend/src/metabase-enterprise/whitelabel/components/IllustrationWidget/IllustrationWidget.tsx

```diff
@@ -135,52 +145,56 @@ export function IllustrationWidget({
           style={{ borderRight: `1px solid ${color("border")}` }}
         >
           {getPreviewImage({
-            value,
-            customSource: settingValues[customIllustrationSetting] ?? undefined,
-            defaultPreviewType,
+            value: value,
+            customSource: customIllustrationSource,
+            defaultPreviewType: type,
           })}
         </Flex>
-        <Flex p="lg" w="25rem" align="center" gap="sm">
+        <Flex p="lg" w="25rem" align="center" gap="md" direction="column">
```

Comment:
the new UI will have a fixed width select and a new line for image upload components instead of having everything on the same line.